src/main/java/com/paneedah/mwc/init/Items.java

```diff
@@ -0,0 +1,13 @@
+package com.paneedah.mwc.init;
+
+import com.paneedah.mwc.bases.ItemBase;
+import com.paneedah.mwc.utils.ModReference;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = ModReference.id)
+public class Items {
+    public static ItemBase sulfur_dust = new ItemBase();
```

Comment:
Missing space between class init and actuall stuff

Reply:
✅